scripts/cita

```diff
@@ -78,7 +78,8 @@ This is the primary script for controlling the $SCRIPT node.
         use "cita append -h" to get more information.
         "cita-config" has the same function.
     port <ports>
-        Sets docker port, for example: "cita port 1337:1337"
+        Sets docker port, for example: "cita port 1337:1337 1338:1338",
```

Comment:
> -p 1337:1337 -p 1338:1338

Should be this?

Reply:
I think `1337:1337 1338:1338` is more friendly to user. They do not need to know the `-p` parameter for docker. And the other way, the cita command `port` is already means `-p`.

Reply:
I see. 
Cause I saw the L-38.